File: gm-utils/src/main/java/src/commom/utils/string/StringEquivalente.java
package src.commom.utils.string;

public class StringEquivalente {

	private StringEquivalente() {}

	public static boolean is(String a, String b) {
		return (
			StringCompare.eqIgnoreCase(
				StringTrim.plus(StringRemoveAcentos.exec(a)),
				StringTrim.plus(StringRemoveAcentos.exec(b))
			)
		);
	}
	
	public static boolean ne(String a, String b) {
		return !is(a, b);
	}

}

File: gm-utils/js/src/commom/utils/string/StringEquivalente.js
import StringCompare from './StringCompare';
import StringRemoveAcentos from './StringRemoveAcentos';
import StringTrim from './StringTrim';

export default class StringEquivalente {

	static is(a, b) {
		return (
			StringCompare.eqIgnoreCase(
				StringTrim.plus(StringRemoveAcentos.exec(a)),
				StringTrim.plus(StringRemoveAcentos.exec(b))
			)
		);
	}

}
